### ISSUE TITLE
Sctructure the training pipeline

### ISSUE THREAD

**Issue by YPolina** [YPolina/News#10]
- [x] Train-Test TimeSeriesSplit
Train-validation-test 70%-20%-10%
- [x] Initialize components (loss function, optimizer, learning rate scheduler
- [x] Run the training loop
- [x] Handle exceptions
